bundles/org.openhab.core.automation/src/main/resources/OH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -231,6 +231,256 @@
 					"reference": "event"
 				}
 			]
+		},
+		{
+			"uid": "core.GroupCommandTrigger",
+			"label": "a member of an item group receives a command",
+			"description": "This triggers the rule if a member of an item group receives a command.",
+			"configDescriptions": [
+				{
+					"name": "groupName",
+					"type": "TEXT",
+					"context": "item",
+					"label": "Group",
+					"description": "the name of the item group",
+					"required": true
+				},
+				{
+					"name": "command",
+					"type": "TEXT",
+					"label": "Command",
+					"description": "the received command",
+					"required": false,
```

Comment:
All other modules specify it explicitly as well and I think it helps for the readability (as it makes clear that this can be null). But if you wish, I can remove it from all module type definitions.

Reply:
Okay, I am fine with that.